vscode-trace-extension/src/utils/trace-server-status.ts

```diff
@@ -0,0 +1,29 @@
+import { StatusBarItem, ThemeColor } from 'vscode';
+import { getTspClient } from './tspClient';
+
+export class TraceServerConnectionStatusService {
+
+    private statusBarItem: StatusBarItem;
+
+    public constructor(statusBarItem: StatusBarItem) {
+        this.statusBarItem = statusBarItem;
+        this.statusBarItem.hide();
+    }
+
+    public async renderStatus(): Promise<void> {
+        const response = await getTspClient().checkHealth();
+        const health = response.getModel();
+        let isUp = false;
+        if (health) {
+            isUp = health.status === 'UP';
+        }
+        if (isUp) {
+            this.statusBarItem.backgroundColor = new ThemeColor('statusBarItem.warningBackground');
+            this.statusBarItem.text = 'Trace Server: Found';
```

Comment:
Good idea; thanks. I'm about to push an improved text as above. As for the tooltip, I'd prefer sticking to a found/not-found terminology, which I'm about to push as well. This is because the server might be online, albeit not found given some misconfiguration on extension's behalf (eventually).